packages/components/src/tree-select/stories/index.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * WordPress dependencies
+ */
+import { withState } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import TreeSelect from '../';
+
+export default { title: 'Components/TreeSelect', component: TreeSelect };
+
+const TreeSelectStory = withState( {
+	page: 'p21',
+} )( ( { page, setState } ) => (
+	<TreeSelect
+		label="Parent page"
```

Comment:
Gutenberg uses a storybook addon called "knobs" that allows the props to be user-editable. It would be awesome if you could add that integration for the label, noOptionLabel, and tree props. The select-control story might be useful as an example that would be pretty similar to this.

https://github.com/WordPress/gutenberg/blob/5fde188a8c3b7603fbb29fca7df56ba34a06b1ae/packages/components/src/select-control/stories/index.js#L33

Reply:
Looks good to me :)